tao/tao_bridge/passes/tao_mark_for_compilation_pass.cc

```diff
@@ -2556,6 +2556,114 @@ std::unordered_set<string>* GetFusedOpTable() {
   return result;
 }
 
+std::vector<string> GetDiscSupportedOps() {
+  std::vector<string> ops;
+  // clang-format off
+  ops.insert(ops.end(), {
+    "Abs",
+    "Add",
+    "AddN",
+    "All",
+    "Any",
+    "BatchMatMul",
+    "BiasAdd",
+    "BiasAddGrad",
+    "BroadcastTo",
+    "Cast",
+    "Ceil",
+    "ConcatV2",
+    "Const",
+    "Conv2D",
+    "DynamicStitch",
+    "Equal",
+    "Erf",
+    "Exp",
+    "ExpandDims",
+    "Fill",
+    "Floor",
+    "FloorDiv",
+    "FloorMod",
+    "GatherNd",
+    "Greater",
+    "GreaterEqual",
+    "Identity",
+    "If",
+    "IsFinite",
+    "LeakyRelu",
+    "Less",
+    "LessEqual",
+    "Log",
+    "LogicalAnd",
+    "LogicalNot",
+    "LogicalOr",
+    "LogSoftmax",
+    "MatMul",
+    "Max",
+    "Maximum",
+    "Mean",
+    "Min",
+    "Minimum",
+    "Mul",
+    "Neg",
+    "NoOp",
+    "NotEqual",
+    "Pack",
+    "Pad",
+    "Pow",
+    "Prod",
+    "RandomUniform",
+    "Range",
+    "RealDiv",
+    "Reciprocal",
+    "Relu",
+    "Relu6",
+    "ReluGrad",
+    "Reshape",
+    "Round",
+    "Rsqrt",
+    "RsqrtGrad",
+    "Select",
+    "SelectV2",
+    "Shape",
+    "Sigmoid",
+    "SigmoidGrad",
+    "Sign",
+    "Size",
+    "Slice",
+    "Snapshot",
+    "Softmax",
+    "SoftmaxCrossEntropyWithLogits",
+    "Softplus",
+    "Split",
+    "Sqrt",
+    "Square",
+    "SquaredDifference",
+    "Squeeze",
+    "StopGradient",
+    "StridedSlice",
+    "Sub",
+    "Sum",
+    "Tanh",
+    "TanhGrad",
+    "Tile",
+    "TopKV2",
+    "Transpose",
+    "Unpack",
+    "While",
+    "ZerosLike"
+  });
+
+#ifndef TF_1_12
+  ops.insert(ops.end(), {
+    "AddV2",
+    "BatchMatMulV2",
+    "GatherV2"
+  });
+#endif
+  // clang-format on
+  return ops;
```

Comment:
`ops` could still be a `static` local right? Ops of 1.12 could still be guarded by the macro.

Reply:
The return value of this function is stored into a static variable. Thus we do not need to use `static` here.